src/sentry/search/events/fields.py

```diff
@@ -1153,9 +1181,25 @@ def normalize(self, value, params):
             if value in {"measurements_value", "span_op_breakdowns_value"}:
                 return ["arrayJoin", [value]]
 
+        if value in {MEASUREMENTS_FRAMES_SLOW_RATE, MEASUREMENTS_FRAMES_FROZEN_RATE}:
+            field = FIELD_ALIASES[value]
+            return field.get_expression(params)
```

Comment:
Is this required? Aren't we able to resolve the expression without this by detecting it's a field alias and adding it to the selected columns?